bin/improver-wind-gust-diagnostic

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Script to create wind-gust data."""
+
+import argparse
+
+import iris
+import warnings
+
+from improver.wind_gust_diagnostic import WindGustDiagnostic
+
+
+def main():
+    """Load in arguments and get going."""
+    parser = argparse.ArgumentParser(
+        description="Calculate revised wind-gust data using a specified "
+        "percentile of wind-gust data and a specified percentile "
+        "of wind-speed data.through the WindGustDiagnostic plugin.")
+    parser.add_argument("input_filegust", metavar="INPUT_FILE_GUST",
+                        help="A path to an input Wind Gust Percentile"
+                        " NetCDF file")
+    parser.add_argument("input_filews", metavar="INPUT_FILE_WINDSPEED",
+                        help="A path to an input Wind Speed Percentile"
+                        " NetCDF file")
+    parser.add_argument("output_filepath", metavar="OUTPUT_FILE",
+                        help="The output path for the processed NetCDF")
+    parser.add_argument("--percentile_gust", metavar="PERCENTILE_GUST",
```

Comment:
Added defaults to these as it is better to be able to distinguish the values between for wind-gust and wind-speed being set in the call to the CLI

Reply:
Thanks for this. As Ben suggests below, these arguments could be extended with `type=float` (https://docs.python.org/2.7/library/argparse.html#type), so that the percentile is automatically a float.